src/plugin.js

```diff
@@ -188,9 +188,12 @@ export const setupSessions = (player) => {
  */
 export const emeErrorHandler = (player) => {
   return (objOrErr) => {
+    const vjsErrorType = videojs.Error && videojs.Error.EMEKeySessionCreationError;
+    const errorType = vjsErrorType || 'eme-key-session-creation-error';
```

Comment:
I guess we should not fallback.
We should expect `videojs.Error.EMEKeySessionCreationError` is available and we should update minimum videojs peerDependency version